fedoracommunity/connectors/bodhiconnector.py

```
@@ -525,13 +525,28 @@ def query_active_releases(self, filters=None, **params):
             filters = dict()
         filters = self._query_updates_filter.filter(filters, conn=self)
         package = filters.get('package')
-        pkgdb = get_connector('pkgdb')
         koji = get_connector('koji')._koji_client
         koji.multicall = True
 
-        for release in pkgdb.get_fedora_releases():
-            tag = release[0]
-            name = release[1]
+        releases_obj = self.call('releases', {})
+        total_pages = releases_obj['pages']
+        releases_all = releases_obj['releases']
```

You should maybe raise an error if you don't get any releases from Bodhi.